commands/auth.go

```diff
@@ -211,7 +211,7 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			printer.PrintError(err.Error())
 			// We don't want usage to be printed as the command was correctly built
```

Comment:
This makes the comment invalid. Let's remove it? Also, does this print the usage? I remember there was a flag to disable printing usage for every single error.

Reply:
Good catch! In fact, we don't need to print the error via printer as well. It's already being printed from command output.